.github/workflows/dist-test.yml

```
@@ -17,19 +17,19 @@ jobs:
   build:
     runs-on: blacksmith-2vcpu-ubuntu-2404
     steps:
-      - uses: actions/checkout@v6
+      - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6
       - name: Install uv
-        uses: astral-sh/setup-uv@v8.1.0
+        uses: astral-sh/setup-uv@08807647e7069bb48b6ef5acd8ec9567f424441b # v8.1.0
         with:
           enable-cache: true
       - name: Set up Python ${{ matrix.python-version }}
```

actionlint (1.7.12)

[error] property "python-version" is not defined in object type {}

(expression)